.github/workflows/release-beta.yml

```
@@ -0,0 +1,49 @@
+    branches:
+      - feat_v3.x
+
+jobs:
+  release-beta:
+#    if: |
+#      startsWith(github.event.head_commit.message, 'chore(release):') &&
+#      contains(github.event.head_commit.message, '-beta.')
+    env:
+      NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+      NPM_CONFIG_PROVENANCE: true
+      TAG: ${{ inputs.tags }}
```

actionlint (1.7.4)

property "tags" is not defined in object type {}

(expression)